quizapp/src/test/groovy/billennium/quizapp/CandidateITSpec.groovy

```diff
@@ -35,6 +49,22 @@ class CandidateITSpec extends Specification {
         response.andExpect(status().isOk())
```

Comment:
`CandidateITSpec` dlaczego `IT` w nazwie

Reply:
IT -  test integracyjny według mojej wiedzy każdy też integracyjny powinien mieć zasze w nazwie  IT